From 1c9ecfe70db04a1a869d22b39f85f1a14e9a017a Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 26 Feb 2020 21:14:52 +0200
Subject: [PATCH 014/230] rework Gray accent overlay v2

	modified:   packages/overlays/AccentColorGrayOverlay/res/values/colors_device_defaults.xml
---
 .../res/values/colors_device_defaults.xml                     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/overlays/AccentColorGrayOverlay/res/values/colors_device_defaults.xml b/packages/overlays/AccentColorGrayOverlay/res/values/colors_device_defaults.xml
index 77e57e86e4e..2d0df98bf17 100644
--- a/packages/overlays/AccentColorGrayOverlay/res/values/colors_device_defaults.xml
+++ b/packages/overlays/AccentColorGrayOverlay/res/values/colors_device_defaults.xml
@@ -17,6 +17,6 @@
  */
 -->
 <resources>
-    <color name="accent_device_default_light">#424242</color>
-    <color name="accent_device_default_dark">#e0e0e0</color>
+    <color name="accent_device_default_light">#808080</color>
+    <color name="accent_device_default_dark">#808080</color>
 </resources>
-- 
2.25.1

